provider/gce/environ_broker_test.go

```diff
@@ -202,7 +202,7 @@ var getDisksTests = []struct {
 
 func (s *environBrokerSuite) TestGetDisks(c *gc.C) {
 	for _, test := range getDisksTests {
-		diskSpecs, err := gce.GetDisks(s.spec, s.StartInstArgs.Constraints, test.Series, "32f7d570-5bac-4b72-b169-250c24a94b2b")
+		diskSpecs, err := gce.GetDisks(s.spec, s.StartInstArgs.Constraints, test.Series, "32f7d570-5bac-4b72-b169-250c24a94b2b", false)
```

Comment:
This looks pretty clear that we need to add another test, or maybe another scenario permutation about how we handle Daily images.
Possibly another test higher up that we notice that "daily" images actually get them from another source? But definitely one at this level that the URL changes if you ask for dailies.